### DISCUSSION TITLE
升级最新内测后，jetbrans相关软件的浮窗弹出都有问题

### DISCUSSION THREAD

**Discussion by debuggerx01** [linuxdeepin/developer-center#9751] [Troubleshooting | 故障排查]
[[问题求助] 升级最新内测后，jetbrans相关软件的浮窗弹出都有问题](https://bbs.deepin.org/post/275109)

【前置条件】：升级v23，7.12内测 ，idea，datagrip等jetbrans系ide
【复现步骤】： 鼠标移到某个可以弹出浮窗的代码api
【预期结果】： 正常看到浮窗
【实际结果】： 浮窗渲染不正常

![image](https://github.com/user-attachments/assets/9f868bad-8b07-4ef1-90da-f95279482de1)

发现可以执行下面的命令临时解决：

`dde-dconfig set org.kde.kwin -r org.kde.kwin.compositing user_type -v 2`
​
效果是新增的弹窗出现动画没了，但是至少IDE的浮窗能正常显示……

看上去新增的动画效果和jetbrans不兼容，希望能够优化解决。

@BLumia 

另外升级后我的机器上使用easystroke这个软件时，屏幕绘制的鼠标手势轨迹起点会出现方块阴影：
![image](https://github.com/user-attachments/assets/17671985-0186-4ac4-abeb-bd01b89b03ad)
需要指定特效类型为4才能解决（`dde-dconfig set org.kde.kwin -r org.kde.kwin.compositing user_type -v 4`），不知是否能够优化

**Comment by BLumia** [comment]
cc @babyfengfjx
